### PR TITLE
Add helm installation

### PR DESCRIPTION
**Pull request by dmitry-shibanov**
- Add installation and validation of helm
Green image (deployed and tested) - https://github.visualstudio.com/virtual-environments/_build/results?buildId=62474&view=results

### CODE REVIEW

File: images/win/scripts/Installers/Validate-Docker.ps1 [hunk 1]
```diff
@@ -44,3 +44,29 @@ _Environment:_
...
+    Write-Host "Short version:"
+    helm version --short
+}
+else
+{
+    Write-Host 'helm is not on path'
+    exit 1
+}
+
+$version = $(helm version --short)
```
Comment (al-cheb) [inline images/win/scripts/Installers/Validate-Docker.ps1]: Duplicate lines 64 and 55

Comment (dmitry-shibanov) [reply images/win/scripts/Installers/Validate-Docker.ps1]: Done.

File: images/win/scripts/Installers/Validate-Docker.ps1 [hunk 2]
```diff
@@ -44,3 +44,27 @@ _Environment:_
 "@
 
 Add-SoftwareDetailsToMarkdown -SoftwareName $SoftwareName -DescriptionMarkdown $Description
+
+# Validate helm
+if (Get-Command -Name 'helm')
+{
+    Write-Host "helm on path"
+    Write-Host "Full version:"
+    helm version
```
Comment (maxim-lobanov) [inline images/win/scripts/Installers/Validate-Docker.ps1]: Should we remove these debug lines?

Comment (dmitry-shibanov) [reply images/win/scripts/Installers/Validate-Docker.ps1]: Done

**Pull request merged**